[2024-05-18T19:19:17.079+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T19:19:17.095+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-05-06T01:00:00+00:00 [queued]>
[2024-05-18T19:19:17.100+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loader.hello_task scheduled__2023-05-06T01:00:00+00:00 [queued]>
[2024-05-18T19:19:17.101+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-18T19:19:17.111+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): hello_task> on 2023-05-06 01:00:00+00:00
[2024-05-18T19:19:17.119+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1247) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-18T19:19:17.120+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_loader', 'hello_task', 'scheduled__2023-05-06T01:00:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/data_loader.py', '--cfg-path', '/tmp/tmprs88d1c0']
[2024-05-18T19:19:17.121+0000] {standard_task_runner.py:63} INFO - Started process 1249 to run task
[2024-05-18T19:19:17.122+0000] {standard_task_runner.py:91} INFO - Job 284: Subtask hello_task
[2024-05-18T19:19:17.163+0000] {task_command.py:426} INFO - Running <TaskInstance: data_loader.hello_task scheduled__2023-05-06T01:00:00+00:00 [running]> on host b5cbc22da81e
[2024-05-18T19:19:17.231+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loader' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-06T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-06T01:00:00+00:00'
[2024-05-18T19:19:17.233+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T19:19:17.234+0000] {logging_mixin.py:188} INFO - 2023-05-06/
[2024-05-18T19:19:17.235+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-18T19:19:17.235+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T19:19:17.242+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_loader, task_id=hello_task, run_id=scheduled__2023-05-06T01:00:00+00:00, execution_date=20230506T010000, start_date=20240518T191917, end_date=20240518T191917
[2024-05-18T19:19:17.255+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-18T19:19:17.268+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T19:19:17.270+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
